internal/build/build.go

```diff
@@ -48,27 +49,42 @@ var configScriptTpl string
 //go:embed templates/k8s_res_deploy.sh.tpl
 var k8sResDeployScriptTpl string
 
-func Run(ctx context.Context, d *image.Definition, buildDir string, system *sys.System) error {
+// nolint:gocyclo
```

Comment:
Here to make the linter happy.

We need to actually freeze the code for the upcoming demos -- a follow up will include the creation of separate Helm handler type to encapsulate the vast majority of the new functionality which in turn would also allow us to drop this suppression.